src/sql/workbench/services/dialog/browser/dialogContainer.component.ts

```diff
@@ -28,7 +28,7 @@ export interface DialogComponentParams extends IBootstrapParams {
 	template: `
 		<div class="dialogContainer" *ngIf="_dialogPane && _dialogPane.displayPageTitle">
 			<div class="dialogModal-wizardHeader" *ngIf="_dialogPane && _dialogPane.displayPageTitle">
```

Comment:
@Benjin @kisantia  I remembered you both ran into issues after changing div to h1/h2, do you remember the details?

Reply:
I had trouble trying to make h1 a live region and ended up using alert to make the wizard page title get announced

Reply:
Yeah - so this isn't going to be announced on page enter or anything. If they go to the element in reading mode then it works but otherwise there's no text announced for that. It's not clear that that's a requirement though - I would expect that the normal usage for people is to use the reading mode if they want to read text.